pulsar-common/src/main/java/com/yahoo/pulsar/common/naming/DestinationDomain.java

```diff
@@ -16,5 +16,29 @@
 package com.yahoo.pulsar.common.naming;
 
 public enum DestinationDomain {
-    persistent
+    persistent("persistent"), non_persistent("non-persistent");
```

Comment:
> But int that case it's part of the path, right? Like http://localhost:8080/admin/non-persistent/.....

Thant's correct, as [HttpLookupService](https://github.com/yahoo/pulsar/blob/master/pulsar-client/src/main/java/com/yahoo/pulsar/client/impl/HttpLookupService.java#L72) creates path with given `destination-name(non-persistent://...)` and it is getting served by [NonPersistentTopics. getPartitionedMetadata servlet](https://github.com/rdhabalia/pulsar/blob/34534370d7bb71fe83175a818090f951824696f2/pulsar-broker/src/main/java/com/yahoo/pulsar/broker/admin/NonPersistentTopics.java#L63)